src/app/Login.tsx

```diff
@@ -1,95 +1,104 @@
 import { useState } from 'react';
+import { useNavigate } from 'react-router-dom'; // useNavigate 추가
 
 import styles from './Login.module.css';
 
 interface LoginProps {
-  goLogin: () => void;
+  goLogin?: () => void;
 }
 
-const Login: React.FC<LoginProps> = ({ goLogin }) => {
+type LoginResponse = {
+  user_id: string;
+  token: string;
+  message: string;
+};
+
+type UserData = {
+  nickname: {
+    nickname: string;
+    tag: string;
+  };
+};
+
+const Login: React.FC<LoginProps> = () => {
```

Comment:
```suggestion
const Login = ({ goLogin }: LoginProps) => {
```
저번에 이렇게 수정하라고 하셨던 듯!